maven-extension/src/main/java/io/opentelemetry/maven/StringUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.maven;
+
+final class StringUtils {
+
+  private StringUtils() {}
+
+  /** Copy org.apache.commons.lang3.StringUtils#isBlank(java.lang.CharSequence) */
+  public static boolean isBlank(final CharSequence cs) {
```

Comment:
Any reason not to use the `String` class methods for these?

Reply:
It's the copy/paste of Commons Lang3 [StringUtils.isBlank(s)](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence- ) with minimum changes. Shall I make it more consistent and use String?

Reply:
Possibly, unless there's a reason to use the Commons Lang3 version I'm not aware of

Reply:
Fixed